internal/workload/manager.go

```diff
@@ -118,6 +118,11 @@ func (w *WorkloadManager) Update(configuration models.DeviceConfigurationMessage
 		return errors
 	}
 
+	errs := w.updateSecrets(configuration)
+	if len(errs) != 0 {
+		errors = multierror.Append(errors, errs...)
```

Comment:
Shouldn't return here? I mean, if secrets cannot be updated, the workloads should not do anything no? @jakub-dzon your input will be great here.

Reply:
You marked as resolved, but no comment or fixes

Reply:
IMO "best effort" approach is good here; there might be workloads that don't need secrets and can be updated/started/removed.

Reply:
make sense :+1: